src/index.ts

```diff
@@ -1,13 +1,2 @@
-import { handlers } from './lib/handlers'
-
-// In order for the workers runtime to find the class that implements
-// our Durable Object namespace, we must export it from the root module.
-export { VoidEventList as EventList } from './lib/implementations/VoidEventList'
-
-export default handlers({
-  interval: 6,
-  functions: {
-    GET: ['getEvents', 'processEvents'],
-    POST: ['setup'],
-  },
-})
+export * from './lib'
+export * from './utils'
```

Comment:
One thing that we could to add is a test folder where we can run the VoidEventList and see if all work